src/app/lib/utilities/preprocessor/rules/addIdsToBlocks.js

```diff
@@ -0,0 +1,48 @@
+import uuid from 'uuid';
+import deepGet from '../../../../helpers/json/deepGet';
+import compose from '../../../../helpers/compose';
+
+let mapIdsToBlocks;
+
+const getJsonContent = jsonRaw => deepGet(['content'], jsonRaw);
+
+const getBlocks = content => deepGet(['model', 'blocks'], content);
```

Comment:
```suggestion
const getBlocks = block => deepGet(['model', 'blocks'], block);
```

`content` is the name of the first block that we encounter in simorgh, however it is semantically just another block. Also because this function is recursively called `content` as a variable name has a different meaning when this function is called by a nested block.